core/templates/dev/head/components/exploration_summary_tile_directive.html

```diff
@@ -4,6 +4,40 @@
       <div class="top-section" style="background-color: <[getThumbnailBgColor()]>;">
         <img class="thumbnail-image" ng-src="<[getThumbnailIconUrl()]>">
         <h2 class="exploration-title protractor-test-exp-summary-tile-title"><[getExplorationTitle()]></h2>
+
+        <div class="exploration-summary-contributors" ng-if="contributors.length > 0"
+             ng-mouseenter="$parent.allContributorsAreVisible = true"
+             ng-mouseleave="$parent.allContributorsAreVisible = false">
+          <profile-link-image username="contributors[0]"
+                              tooltip="<[contributors[0]]>"
+                              tooltip-placement="left"
+                              tooltip-append-to-body="true">
+          </profile-link-image>
+          <p ng-show="!allContributorsAreVisible && contributors.length > 1"
+             class="exploration-committers-number-1">
+            +<[contributors.length - 1]>
+          </p>
+
+          <div ng-show="allContributorsAreVisible">
+            <profile-link-image ng-repeat="contributor in contributors | limitTo:4:1"
+                                username="contributor"
+                                tooltip="<[contributor]>"
+                                tooltip-placement="left"
+                                tooltip-append-to-body="true">
+            </profile-link-image>
+
+            <div style="position:relative;" ng-show="contributors.length > 5">
```

Comment:
Can you pull the number 5 out into a $scope constant? (E.g. $scope.MAX_CONTRIBUTORS_TO_DISPLAY)